### DISCUSSION TITLE
Fabfile missing

### DISCUSSION THREAD

**Discussion by mmansour** [stephenmcd/mezzanine#2024] [General]
Hi All, 

Sorry if I missed this. Is mezzanine no shipping with a fab file?

**Comment by jerivas** [comment]
It is not as of [v5.0.0](https://github.com/stephenmcd/mezzanine/releases/tag/v5.0.0) (see Breaking Changes)

**Comment by mmansour** [comment]
God thanks for the quick response.

So there's no more deployment support from messaging? Just figure it out
Django style?

**Reply by jerivas** [reply]
That's correct

**Comment by mmansour** [comment]
Please excuse the voice to text auto correct :))

**Comment by mmansour** [comment]
Got it.  Thanks again for the quick response and for all the great work on
the project.

**Comment by molokov** [comment]
For several years now, I've used Fabric to deploy my site (and kept an older fabfile when I moved to Mezzanine 5.0.0). I'm now in the process of upgrading to Mezzanine 6.0.0, Django 4.x and Python 3.8+ (which actually required me to update my OS from Ubuntu 18.04 to 20.04!)

It's probably a good idea to now also deploy using WSGI/ASGI given that's the approved Django method.

So... simple and probably stupid question... how do I actually do this? The Django docs indicate how to set up the wsgi.py and your settings, but there's no actual "command" to run which deploys from my dev machine to the server.

So my question is: What's the equivalent of `fab deploy` that I need to learn to use?

**Comment by mmansour** [comment]
I ended up finally investing time into Docker.

But I still miss the fab approach.

**Comment by jerivas** [comment]
For anyone also looking for deployment advice, I suggest you check out these hosting providers that have tutorials for Django applications (of which Mezzanine is one) and decent free tiers:

- [Render](https://render.com/docs/deploy-django)
- [Fly.io](https://learndjango.com/tutorials/django-hello-world-flyio-deployment)
- [Python Anywhere](https://help.pythonanywhere.com/pages/DeployExistingDjangoProject/)
